site/content/protocols/mediator-coordination/2.0/readme.md

```diff
@@ -180,18 +180,18 @@ where:
 
 - `paginate`: is optional, and if present must include `limit` and `offset`.
 
-### Keylist 
-Response to key list query, containing retrieved keys.
+### Didlist 
```

Comment:
Should tbis be recipient list?

Reply:
Looks like I missed that. Yes, it should be, nice catch!

Reply:
Changed "Didlist" to "Recipient" and updated the following line 184 to match